File: 26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        duplicates = []
        for i in range(len(nums)):
            if len(duplicates) == 0 or nums[i] != duplicates[len(duplicates) - 1]:
                duplicates.append(nums[i])
        nums.clear()
        for i in range(len(duplicates)):
            nums.append(duplicates[i])
        return len(duplicates)
        

File: 26_remove_duplicates_from_sorted_array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        std::vector<int> duplicates = {};
        for (int i = 0; i < nums.size(); i++) {
            if (duplicates.size() == 0 or nums[i] != duplicates[duplicates.size() - 1]) {
                duplicates.push_back(nums[i]);
            }
        }
        nums.clear();
        for (int i = 0; i < duplicates.size(); i++) {
            nums.push_back(duplicates[i]);
        }
        return duplicates.size();
    }
};